frameworks/Java/firenio/src/main/java/hello/TestHttpLoadServer.java

```diff
@@ -133,6 +134,12 @@ public void channelOpened(Channel ch) throws Exception {
                 }
             });
         }
+        ByteTree cachedUrls = null;
+        if (cachedurl){
```

Comment:
Could you explain a little bit what `cachedUrls` is?

Reply:
It used to cut down String instance. For example when decode the first line of http protocol, the url may always in the server configured path, so no more “new String(url bytes)” is needed.